lib/mindwendel_web/controllers/static_page_html/home.html.heex

```diff
@@ -1,10 +1,5 @@
 <div class="d-flex w-100 h-100 p-3 pb-0 mx-auto flex-column bg-mindwendel text-white">
   <header class="mb-auto bg-none container">
-    <img
-      src={Routes.static_path(@conn, "/images/mindwendel_logo_white.svg")}
-      height="40px"
-      class="float-md-start me-3"
-    />
```

Comment:
que? (why did you remove the image without replacement?)

Reply:
needs to be added back in. everything content wise that was removed was just temporary so I could temporarily remove one more error message. in the end all content should be in again. Guess it was during my testing to get the new route feature working.